dev-tools.sh

```diff
@@ -34,5 +34,6 @@
 #
 # Uncomment and set the Dev-Tools's commit value and commit this change.
 # export GH_COMMIT=COMMIT_SHA
+export GH_COMMIT=ed88053154aaf46c43a5289b126621242d01afe8
```

Comment:
revert this?

Reply:
yup, once [this has been](https://github.com/dpc-sdp/dev-tools/pull/80) approved and released I will do that.

Reply:
Ah, this is the way.